server/crawler/crawler.go

```diff
@@ -233,28 +235,41 @@ func (crawler *GithubCrawler) crawlKurtosisPackages(
 	return len(kurtosisPackageUpdated), len(kurtosisPackageAdded), len(kurtosisPackageRemoved), nil
 }
 
-func ReadPackage(
+func (crawler *GithubCrawler) ReadPackage(
```

Comment:
@leoporoli What's the need for making this a function that depends on the crawler? We don't use the crawler in this context?

Reply:
Well, I did it because I was planning to use the `crawler.store` to get the repository from the store but I'll revert it now that I won't use the store here